code-challenges.test.js

```diff
@@ -16,52 +16,232 @@
+
+        const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+
+        expect(toCapitolize(randomNouns1)).toEqual(["STREETLAMP", "POTATO" , "TEETH" , "CONCLUSION" , "NEPHEW"])
+        expect(toCapitolize(randomNouns2)).toEqual(["TEMPERATURE" , "DATABASE" , "CHOPSTICKS" , "MANGO"]);
+        })
+})
+
+const toCapitolize = (arr) => {
+    return arr.map(word => word.toUpperCase())
+    }
+//    Testing:
+//    console.log(toCapitolize(randomNouns1)) // output: [ 'STREETLAMP', 'POTATO', 'TEETH', 'CONCLUSION', 'NEPHEW' ]
+//    console.log(toCapitolize(randomNouns2)) // output: [ 'TEMPERATURE', 'DATABASE', 'CHOPSTICKS', 'MANGO' ]
+   
+
+
+    // PASS  ./code-challenges.test.js
+    // toCapitolize
+    //   ✓ takes in an array of strings (1 ms)
+        
 
 // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
 
+
 // Pseudo code:
+// input: takes in a string 
+// output:logs the index [i] of the first vowel 
+// How? .indexOf(), using a for loop to iterate through(might be able to use .map())
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const vowelTester1 = "learn"
+//   describe("firstVowelIndex" , () => {
+//     it("takes in a string" , () => {
+//         const vowelTester1 = "learn"
+//         const vowelTester2 = "academy"
+//         const vowelTester3 = "challenges"
+//         expect(firstVowelIndex(vowelTester1)).toEqual(1);
+//         expect(firstVowelIndex(vowelTester2)).toEqual(0);
+//         expect(firstVowelIndex(vowelTester3)).toEqual(2)
+//         })
+// })
+
+// FAIL  ./code-challenges.test.js
+// firstVowelIndex
+//   ✕ takes in a string (1 ms)
+
+// ● firstVowelIndex › takes in a string
+
+//   ReferenceError: firstVowelIndex is not defined
+
+// const vowelTester1 = "learn"
 // Expected output: 1
-const vowelTester2 = "academy"
+// const vowelTester2 = "academy"
 // Expected output: 0
-const vowelTester3 = "challenges"
+// const vowelTester3 = "challenges"
 // Expected output: 2
 
 // b) Create the function that makes the test pass.
+
+  describe("firstVowelIndex" , () => {
+    it("takes in a string and logs the index of the first vowel" , () => {
+        const vowelTester1 = "learn"
+        const vowelTester2 = "academy"
+        const vowelTester3 = "challenges"
+        expect(firstVowelIndex(vowelTester1)).toEqual(1);
+        expect(firstVowelIndex(vowelTester2)).toEqual(0);
+        expect(firstVowelIndex(vowelTester3)).toEqual(2)
+        })
+})
+
+            
+// const vowelTester1 = "learn"
+// const vowelTester2 = "academy"
+// const vowelTester3 = "challenges"
+
+
+
+  const firstVowelIndex = (str) => {
+    const vowels = ["a", "e", "i", "o", "u"];
+    for (let i = 0; i < str.length; i++) {
+      if (vowels.includes(str[i])) { // you can add .toLowerCase() for instances of a Capitalized vowel.
+        return i;
+      }
+    }
+    return -1; // for when no vowels are found
```

Comment:
I'm a big fan of the pseudo code next to each line! However, in your original pseudo code on lines 174 - 176, you did not mention creating an array to hold vowels, or using `.includes`. You also did not explain what this code was actually doing.